app/controllers/Authentication.scala

```diff
@@ -531,7 +534,13 @@ class Authentication @Inject()(actorSystem: ActorSystem,
   private def checkOrganizationFolder(implicit request: RequestHeader) =
     for {
       datastores <- dataStoreDAO.findAll(GlobalAccessContext)
-      _ <- Fox.serialCombined(datastores)(d => rpc(s"${d.url}/data/triggers/checkNewOrganizationFolder").get)
+      _ <- Fox.serialCombined(datastores)(d =>
+        rpc(s"${d.url}/data/triggers/checkNewOrganizationFolder").get.futureBox.map {
+          case Full(_) => Fox.successful(())
+          // This failure exists if the datastore is down. We still want to be able to create organizations, therefore we ignore this error.
+          case Failure(errorMsg, _, _) if errorMsg.contains("Connection refused") => Fox.successful(())
```

Comment:
I think it is acceptable to remove this checking method and route netirely, as this part-ignoring of the result of these things are going to create confusion. I’d say _no_ folder-creation issues should block the registration process. Instead, could you add a readable error message to the dataset upload route (where the directory needs to be created if it is missing), in case that fails due to permission issues?